src/components/ha-date-range-picker.ts

```diff
@@ -168,28 +170,6 @@ export class HaDateRangePicker extends LitElement {
                   }
                 ),
               ],
-              [this.hass.localize(
-                "ui.components.date-range-picker.ranges.last_month"
```

Comment:
Just because we have a previous button, I still think this has value

Reply:
As described, the idea was to save space for mobiles, as on my mobile the whole list does not fit.

Reply:
@bramkragten I also think we should remove redundant presets. We can remove them only on mobile but this would add complexity and we'll get bug reports